metricpublisher/schemas/log_event.json

```diff
@@ -0,0 +1,162 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Schema for the log_event lambda request object",
+    "type": "object",
+    "definitions": {
+        "request_id": {
```

Comment:
Only include the attributes have "items" or "properties" under them in "definitions". Simple ones like "request_id" does not need to have separate definitions.

Reply:
Oh got ya. I thought you meant to have separate definitions for the properties themselves, not the ones that have properties underneath them...